src/PrestaShopBundle/Routing/YamlModuleLoader.php

```diff
@@ -82,4 +84,24 @@ public function supports($resource, $type = null)
     {
         return 'module' === $type;
     }
+
+    /**
+     * @param RouteCollection $routes
+     *
+     * @return RouteCollection
+     */
+    private function modifyRoutes(RouteCollection $routes)
+    {
+        foreach ($routes->getIterator() as $route) {
+            if ($route->hasDefault('_disable_module_prefix')) {
+                if ($route->getDefault('_disable_module_prefix') === true) {
```

Comment:
Why double `if` statement ? 
Use `&&` operator.

Reply:
done

Reply:
Indeed 🤔

Reply:
Thanks @matthieu-rolland for fixing my branch 😉